pom.xml

```diff
@@ -473,6 +473,16 @@
         <artifactId>aws-java-sdk</artifactId>
```

Comment:
side note: Github is doing a really unfortunate thing and showing your old comments next to my new diffs